OpenSim/Simulation/Model/Geometry.h

```diff
@@ -155,12 +144,12 @@ class OSIMSIMULATION_API Geometry : public Component {
     };
 
     /// Convenient access to set Appearance/representation
-    void setRepresentation(const DisplayPreference& rep) { 
+    void setRepresentation(DecorativeGeometry::Representation rep) { 
```

Comment:
Comment also seems incorrect. Since it is the 'DecorativeGeometry::Representation' that is being set. Unless is is also the same thing as the `SurfaceAppearance` `representation` (property) , which is confusing.